coordinator/src/node/expired_positions.rs

```diff
@@ -57,10 +58,7 @@ pub async fn close(node: Node) {
         };
 
         let closing_price = match BitmexClient::get_quote(&position.expiry_timestamp).await {
```

Comment:
I know it's not your code, but this price should not be from bitmex but from our orderbook.

Reply:
I see where you are going, but for me using a price from the orderbook only makes sense if there is also an order that is eventually executed. We don't do this at the moment.
The coordinator can still use an orderbook price and just simulate the match - but I find this a bit odd. 
I opened https://github.com/get10101/10101/issues/988 to capture this; feel free to change the description